app/victoria-traces/main.go

```diff
@@ -49,6 +50,9 @@ func main() {
 	insertutil.SetLogRowsStorage(&vtstorage.Storage{})
 	vtinsert.Init()
 
+	// optional background task(s)
+	servicegraph.Init()
```

Comment:
I don’t think this comment adds much value; consider removing it or adding more context.